deploy/olm-catalog/windows-machine-config-operator/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -4,6 +4,11 @@ metadata:
   annotations:
     alm-examples: '[]'
     capabilities: Basic Install
+    categories: OpenShift Optional
+    certified: "false"
+    containerImage: REPLACE_IMAGE
+    createdAt: "2020-12-08T12:14:00Z"
```

Comment:
Are you sure there is no way to get this filled using `render_templates` using some env var?

Reply:
adding a REPLACE_DATE field for this to be changed through render_templates script.